disk_usage/disk_usage.py

```diff
@@ -27,40 +28,33 @@
 
 
 def aggregate_level(name: str) -> str:
-    """Returns a prefix for the given blob name at the aggregation level."""
+    """Returns a prefix for the given blob name at the folder or Hail table level."""
```

Comment:
Should throw a couple docstring tests in here :)

Reply:
Thanks, done!